bats/README.md

```diff
@@ -0,0 +1,143 @@
+## Overview
+
+BATS is a testing framework for Bash shell scripts that provides supporting libraries and helpers for customizable server-side test automation.
+
+## Setup
+
+It's important to have a Rancher Desktop CI build installed with privileged rights before running the BATS tests.
+
+On Windows:
+  Prior to running `git submodule update --init`, it's important to setup the Git configuration by running the following commands:
+
```

Comment:
I think you should explain that you need to clone the repo first.

Note that changing crlf settings is not necessary when you clone inside a WSL distro; it is only needed when you clone on Win32.